src/main/java/com/minecolonies/coremod/colony/buildings/BuildingBuilder.java

```diff
@@ -165,6 +176,95 @@ public void writeToNBT(@NotNull final NBTTagCompound compound)
     }
 
     /**
+     * Hold the pair of number of avaliable and needed StackItem
+     */
+    public static class BuildingBuilderResource
+    {
+        public enum RessourceAvailability
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Document this public enum. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)